### PR TITLE
add pairs and test to repo

### PR DESCRIPTION
**Pull request by pmm09c**
Add support for polygon, avalanche, and bsc marketplace pairs

### CODE REVIEW

File: src/dextrader/nomics/pairs.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+nomics = {"chain": {}}
+
```
Comment (sububer) [inline src/dextrader/nomics/pairs.py:2]: now that this is starting to gel for me, was curious if tomorrow you could walk through the process of discovering/finding the hash value for a particular chaing/exchange/market.

File: src/nomics_tests.py [hunk 2]
```diff
@@ -33,7 +34,22 @@ def test_candles():
...
 
+def test_pairs():
+    for chain_id in nomics["chain"].keys():
+        for exchange in nomics["chain"][chain_id]["exchanges"]:
+            for market in nomics["chain"][chain_id]["exchanges"][exchange]["markets"].keys():
+                hash = nomics["chain"][chain_id]["exchanges"][exchange]["markets"][market]
+                print(chain_id, exchange, market, hash)
+    start = "2022-01-18"
+    end = "2022-01-19"
+    query = get_candles(exchange, hash, start, end)
```
Comment (sububer) [inline src/nomics_tests.py:45]: what does this do? `query` will always only contain the reslts of a single call to get_candles with whatever happens to be the last value of `exchange` and `hash` after executing the above loops. is that the intent?

**Review by sububer — APPROVED** [review]
curious about the intent of the query in the test, but looks good. thanks for building that dict of chain/exchange/market lookups

File: src/nomics_tests.py (hunk 2 above)
Comment (pmm09c) [reply src/nomics_tests.py:45]: Ahh, i didn't want to spend too many queries doing it for all of them, so really it's just dumping out the dictionary and checking the last one didn't give an error.

**Pull request merged**